data/human/hails.txt

```diff
@@ -1803,6 +1882,58 @@ phrase "friendly navy"
 		"in the Republic"
 	word
 		"!"
+		
+phrase "friendly navy"
```

Comment:
I think this phrase might need some changing. Two things:
1. It doesn't really feel like the second sentence logically follows the first. The person is talking about how sacrificing life and limb is honorable, then switches to strategy against hostiles. Those two things are marginally related, sure, but there's not really a transition from one to the other. I'd suggest separating these two sentences into two separate `phrase`s.
2. I'm not sure if it's really that important, but the potential combinations of `word`s mean that you can hear somewhat contradictory ideas. One officer might tell you that capturing pirates is preferable, and another might tell you killing them is. Maybe it could be edited to make it clear that these are opinions of the individual officers rather than their orders?

Reply:
Those texts were inspired by a quote from general George S. Patton; "The object of war is not to die for your country but to make the other bastard die for his." It seems to sound more related in Pattons quote than in the texts here.

Killing pirates and capturing pirates are both about making sure that they no longer pose a thread, so the generic idea behind such orders doesn't sound very contradictionary. But I can make it sound like an opinion (by adding texts like "but in my opinion", "but if you ask me" to connect the phrases).

Reply:
I updated the text a bit and I hope I made the connection also more explicit. Please let me know if it works.

Reply:
Yeah, I think the transition works much better now!